lib/Re2.fx

```diff
@@ -0,0 +1,1234 @@
+/*
+    This file is a part of ficus language project.
+    See ficus/LICENSE for the licensing terms
+*/
+
+/*
+    Re2 library wrapper.
+
+    Usage: 
+    1.) Re2 library must be installed in system, see installation instructions there:
+    https://github.com/google/re2/wiki/Install
+
+    Sometimes(E.g.,, on Debian 10) it's also needed to run
+        sudo ldconfig 
+    after installation for creating appropriate soft links for shared libraries.
+
+    2.) Before compilation set environment variable FICUS_LINK_LIBRARIES to "-lre2" for providing compiler info about dependencies.
+        e.g, in bash:
+        export FICUS_LINK_LIBRARIES="-lre2"
+    3.) Use -c++ flag on compilation.
+
+
+    See https://github.com/google/re2/wiki/Syntax re2.h of Re2 source file for more detailed information about syntax and options. 
+    Wrapper function signatures have ficus-dependent specialization. They are described there.
+*/
+
```

Comment:
insert
```
pragma "c++", "clib:re2"
```
here.
after it, if "re2/re2.h" header is in the path and "libre2.a" is the path too (e.g. in `/usr/local/{include,lib}`), the example can be run with simple `ficus -run ../examples/regexp.fx`. It will automatically compile just Re2.fx with C++ compiler and link re2 library to the binary

Reply:
Done.